draft-ietf-mls-protocol.md

```diff
@@ -2965,6 +2965,7 @@ following structure, comprising all the fields in the PublicGroupState above
 
 ~~~~~
 struct {
+    CipherSuite cipher_suite;
```

Comment:
```suggestion
    ProtocolVersion version = mls10;
    CipherSuite cipher_suite;
```

Reply:
Can't the ProtocolVersion be inferred from the Ciphersuite?

Reply:
No.  That's why they're separate fields in KeyPackage.